docs-src/concepts/what-is-persistence.md

```diff
@@ -0,0 +1,46 @@
+---
+id: what-is-persistence
+title: What is Persistence?
+sidebar_label: Persistence
+description: Temporal Persistence store is a database used by Temporal Services to persist events generated and processed in the Temporal Cluster and SDK.
+tags:
+  - term
+  - explanation
+---
+
+Temporal Persistence store is a database used by [Temporal Services](/concepts/what-is-the-temporal-server) to persist events generated and processed in your Temporal Cluster and SDK.
```

Comment:
two databases? core and visibility are separated.

Reply:
This section refers to the core database; I have a separate topic that discussed Visibility.